docs/serverless/03-02-function-specification.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Function's specification
+type: Details
+---
+
+Serverless in Kyma allows you to create Functions in both Node.js (10 & 12) and Python (3.8). Although the Function's interface is unified, its specification differs depending on the runtime used to run the Function.
+
+## Signature
+
+Function's code is represented by a handler that is a method processing events. When your Function is invoked, it runs this handler method to request event details and return a given response.
```

Comment:
```suggestion
Function's code is represented by a handler that is a method of processing events. When your Function is invoked, it runs this handler method to request event details and return a given response.
```

Reply:
I meant `method` as in programming so I will leave it as it is.